src/Symfony/Component/PropertyInfo/Tests/PropertyInfoCacheExtractorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo\Tests;
+
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\PropertyInfo\PropertyInfoCacheExtractor;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PropertyInfoCacheExtractorTest extends AbstractPropertyInfoExtractorTest
+{
+    public function setUp()
```

Comment:
Quick search revealed there's few more occurrences.  I'll send a PR to fix them.